WaterLog_Backend/Controllers/TankLevelsController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using WaterLog_Backend.Models;
+
+namespace WaterLog_Backend.Controllers
+{
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TankLevelsController : ControllerBase
+    {
```

Comment:
You should be returning action results on all of these functions i.e `Ok();`, `NotFound();`

Reply:
done